dashboard/modules/job/pydantic_models.py

```diff
@@ -98,3 +98,7 @@ class JobDetails(BaseModel):
         None,
         description="The node ID of the node the job entrypoint command is running on.",
     )
+    driver_exit_code: Optional[str] = Field(
```

Comment:
Out of curiosity why Optional[str] and not Optional[int]? Also, in the description we should also define when it would return `None`.

Reply:
good catch, the actual output is int so shld update that